nevergrad/optimization/optimizerlib.py

```diff
@@ -1918,6 +1918,7 @@ def __init__(
 chainNaiveTBPSACMAPowell.no_parallelization = True
 
 # new names
+CGDE = Chaining([RotatedTwoPointsDE, TwoPointsDE], [200]).set_name("CGDE", register=True)
```

Comment:
experimental variants for now?
and why not a more explicit name than an acronym?

Reply:
It's the best performing algorithm on several benchmarks, and the main algorithm for reporting good results from one part of the genome to another. I would go for inclusion in the main optimizerlib.py
I update the name.